src/JupyterHub.ts

```diff
@@ -15,6 +16,16 @@ class JupyterHub {
         var t = this._decodeToken(token)
         const protocols = ['https', 'http']
         var user = undefined
+        var hosts =  whitelistConfig.whitelist
```

Comment:
What if the user don't have this config? You might need to add default value like `whitelistConfig.whitelist ?? []`

Reply:
Done !